src/react-components/profile-info-header.js

```diff
@@ -1,19 +1,35 @@
 import React from "react";
 import PropTypes from "prop-types";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+import faQuestion from "@fortawesome/fontawesome-free-solid/faQuestion";
+import faShareAlt from "@fortawesome/fontawesome-free-solid/faShareAlt";
 
 export const ProfileInfoHeader = props => (
   <div className="profile-info-header">
-    <img src="../assets/images/account.svg" onClick={props.onClick} className="profile-info-header__icon" />
-    <div className="profile-info-header__profile_display_name" onClick={props.onClick} title={props.name}>
-      {props.name}
+    <div className="profile-info-header__profile_display_name">
+      <img src="../assets/images/account.svg" onClick={props.onClickName} className="profile-info-header__icon" />
+      <div onClick={props.onClickName} title={props.name}>
+        {props.name}
+      </div>
```

Comment:
Long names and the account image are right-aligned and cut off now.
![image](https://user-images.githubusercontent.com/79419/39099876-5df4ce64-4636-11e8-8877-934f56b72f69.png)